Autopilot/SensorFusion/Inc/sensorFusionInterface.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SENSORFUSIONINTERFACE_H
+#define SENSORFUSIONINTERFACE_H
+
```

Comment:
Duplicate file

Reply:
I say we should stick with sensorFusionInterface.hpp

Reply:
Sounds good, I think I forgot to delete this one before committing